### ISSUE TITLE
Option to strip quote marks from quoted args

### ISSUE THREAD

**Issue by joeflateau** [binocarlos/spawn-args#3]
I use this to launch ffmpeg, ffmpeg doesn't like the quote marks around arguments, could there be an option to remove the quote marks around a quoted arg? I fixed for now by:

```
args.map(function(arg){ return arg.replace(/^"|"$/g, ""); });
```

**Comment by binocarlos** [comment]
So I just pushed 0.1.0 to npm - the parse function now takes a second argument that is an options object.

Currently there is one option **removequotes** - all it does is [your map function](https://github.com/binocarlos/spawn-args/blob/master/index.js#L68) with one caveat - it keeps the quotes for values that have spaces (otherwise it would become 2 values when parsed by a shell)

You can see this in action in [the test](https://github.com/binocarlos/spawn-args/blob/master/test/test.js#L39)

It would be really great if you could make a pull request with an updated Readme containing some docs for this :-)

**Comment by dominicbarnes** [comment]
Can this behavior also strip single quotes?

Can we add an option to remove the outer quotes regardless? I'm using this to build a CLI-like tool, and I would like it to behave like `process.argv`:

```js
console.log(process.argv);
```

```sh
$ node index.js
[ 'node', 'index.js' ]

$ node index.js "hello world"
[ 'node', 'index.js', 'hello world' ]

$ node index.js --message "hello world"
[ 'node', 'index.js', '--message', 'hello world' ]
```

**Comment by binocarlos** [comment]
@dominicbarnes thanks for your message.

> Can this behavior also strip single quotes?

Yes this would be a useful Pull Request :-)

> Can we add an option to remove the outer quotes regardless?

I can't quite see what you mean (a.k.a. I am being stupid) - can you elaborate on what you mean by outer quotes?

Thanks!

**Comment by dominicbarnes** [comment]
I'll do my best to explain further.

When adding quotes to a CLI command, typically it's only there to provide clarity. In other words, if my argument has spaces, without quotes it wouldn't know how to group the argument.

```
mv "Hello World.txt" hello-world.txt
```

The file name in question doesn't have quotes in it, but the quotes are a clue to the CLI that the argument is grouped together.

Basically, I propose that the `removequotes` parameter _always_ strips the quotes, even if there are spaces. In my case, I never want those quotes around. Another option would be to add `removequotes: 'always'`, if you still think the current behavior is useful.

**Comment by binocarlos** [comment]
@dominicbarnes So I've just pushed the [remove-quotes-always](https://github.com/binocarlos/spawn-args/tree/remove-quotes-always) branch.

If you could check if this is what you are after I'll marge to master, bump the version and `npm publish`

**Comment by sebakerckhof** [comment]
+1. The remove quotes always branch also solves my problem. So it'd be great to see it merged in.

**Comment by binocarlos** [comment]
@sebakerckhof hey thanks - I ha forgotten about this! I've just merged that branch and pushed `0.2.0` to npm

**Comment by sebakerckhof** [comment]
Great thanks for the fast response.
